### ISSUE TITLE
YoloR-CSP Test with yolor-csp.cfg not working

### ISSUE THREAD

**Issue by GabrielFerrante** [WongKinYiu/yolor#281]
Hi everyone

I'm using the command for test in my model trained. 

python3 test.py --data ./BRA-Dataset.yaml --img 412 --batch 8  --device 0 --cfg cfg/yolor_csp.cfg --weights ../../PESOS1/bestYoloR-CSP.pt --name yolor_csp_val --verbose --names data/BRA.names

I'm configured the yolor_csp.cfg for test, modifying the filters for 30 (num classes(5) + 5 * 3), the number classes 5 and, implicit_mul with 30.

But I'm not have a Precision, Recall and, small mAP. However, while I executing test for yolor-p6 model, working not problems. 

The csp.cfg working ? I see that the csp.cfg not have a YoloR layer in final part of file cfg. Foremore, the csp.cfg have a 3 implict_mul, different in comparison with p6.cfg. 

My Output:
Model Summary: 529 layers, 52519444 parameters, 52519444 gradients
WARNING: --img-size 412 must be multiple of max stride 64, updating to 448
/home/usp/anaconda3/envs/yoloEnv/lib/python3.8/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
Scanning labels ../../BRA-Dataset/labels/val.cache3 (363 found, 0 missing, 0 empty, 0 duplicate, for 363 images): 363it [00:00, 16860.82it/s]
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95:  50%|██████████████████████████████                              | 23/46 [00:02<00:01, 11.95it/s]libpng warning: iCCP: known incorrect sRGB profile
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████████████| 46/46 [00:04<00:00, 10.71it/s]
                 all         363         403           0           0     0.00315    0.000539
                Anta         363          84           0           0     0.00095    0.000168
          Jaguarundi         363          68           0           0     0.00144    0.000282
           LoboGuara         363          82           0           0     0.00157    0.000302
           OncaParda         363         101           0           0     0.00474    0.000945
    TamanduaBandeira         363          68           0           0     0.00704    0.000998
Speed: 6.5/2.9/9.5 ms inference/NMS/total per 448x448 image at batch-size 8
Results saved to runs/test/yolor_csp_val2
